### DISCUSSION TITLE
React Floater prop `portalElement` not supported

### DISCUSSION THREAD

**Discussion by pankajpatel** [gilbarbara/react-joyride#1008] [Q&A] [answered]
In my current setup, automatic portal is not working as the setup involved iframes and hiding many elements adjacent to iframe inside the body.

Any element/component is mounted to a div under `html`

When using joyride, all the steps are mounted to a portal inside body which cause it to be not shown properly.

I checked the `react-floater@0.9.3` supports a prop named `portalElement` to allow attaching steps in other places. This can help my current setup but in `react-joyride`, `react-floater@0.7.9` is used and it does not install 0.9.3

Any suggestions on forcing the use of `react-floater@0.9.3`?

**Comment by gilbarbara** [comment, marked as answer]
Hey @pankajpatel 
Unfortunately, it's not possible.
The react-floater 0.9.x isn't compatible with react-joyride yet.
